chart/templates/service_discovery_controller.yaml

```diff
@@ -1,3 +1,4 @@
+{{- include "_config.load" $ }}
```

Comment:
I think I mentioned this before - can we lint for this?

Reply:
Yes, there is a plan for this somewhere; might still be in the multi-stack issue. Either way, I'm aware and will file an issue for it if we don't have one yet.